kadane_algorithm/kadane.cpp

```diff
@@ -0,0 +1,34 @@
+#include <array>
+#include <climits>
+#include <iostream>
+
+int maxSubArray(const std::array<int, 5> &n) {
```

Comment:
Just the documentation template.

Provide brief information about the
function, parameters and return statement.

```suggestion
/**
 * @brief description
 * @param n description
 * @returns description
 */
int maxSubArray(const std::array<int, 5> &n) {
```

Reply:
See #962 for how to document functions.